controller/volume_controller.go

```diff
@@ -2805,6 +2829,60 @@ func (c *VolumeController) updateRequestedBackupForVolumeRestore(v *longhorn.Vol
 	return nil
 }
 
+func (c *VolumeController) checkAndInitVolumeOfflineReplicaRebuilding(v *longhorn.Volume, rs map[string]*longhorn.Replica) error {
+	log := getLoggerForVolume(c.logger, v)
+
+	if v.Spec.BackendStoreDriver == longhorn.BackendStoreDriverTypeLonghorn {
+		return nil
+	}
```

Comment:
Won't we provide this feature for legacy volumes?

Reply:
Yes, we can extend the feature to legacy volumes as I mentioned in the ticket.
Not sure if there is any unknown impact on the legacy volumes, so just put it in the SPDK volumes first.